tpls/gtest/gtest/gtest.h

```diff
@@ -4910,7 +4910,7 @@ class NeverThrown {
   class GTEST_TEST_CLASS_NAME_(test_suite_name, test_name)                    \
       : public parent_class {                                                 \
    public:                                                                    \
-    GTEST_TEST_CLASS_NAME_(test_suite_name, test_name)() = default;           \
+    GTEST_TEST_CLASS_NAME_(test_suite_name, test_name)() { (void)test_info_; }\
```

Comment:
Could you please reference issues or rejected prs on googletest that show that there is no interest on resolving this upstream?

Reply:
https://github.com/google/googletest/issues/4104#issuecomment-1376301742 mentions that `nvcc` is an unsupported compiler (in a slightly different context).

Reply:
> Could it be a problem if we later want to update gtest ?

We will notice if this is still a problem or not.

> What do you think of pre-installing all Kokkos dependencies in the CI so that we use the flags we want solely on Kokkos ? We could then compile gtest with the host compiler ?

This is not about installing `gtest` but rather using it. We compiler warnings when we define unit tests.

Reply:
> This is not about installing `gtest` but rather using it. We compiler warnings when we define unit tests.

I had in mind that using an installed gtest would treat gtest include header path as system path thus removing the warnings ?

Reply:
We could try that (in a separate pull request); it's certainly a bigger change than changing this one line.